.travis.yml

```diff
@@ -28,46 +28,72 @@ cache:
 
 matrix:
   include:
-    - python: 2.7
+    - stage: static
+      python: 2.7
+      env: TOX_ENV=py27-static
+    -
```

Comment:
not sure i understand the convention of blank lines in here

Reply:
It is a bit weird. Was following https://blog.travis-ci.com/2017-05-11-introducing-build-stages .

Re-reading that suggests that they're _optional_ though, so I will put them back in again.